src/VM/OpcodeHandler.h

```diff
@@ -27,6 +27,9 @@
 
 // Third party includes
 
+// Marks handler parameter as unused and prevents compiler warnings.
+#define UNUSED_ARG(arg) (void)(arg)
```

Comment:
I mean that they can be considered as some sort of arguments for OpcodeHandlers.
Removed the macro and pushed new version.